ideas/145-identity.md

```diff
@@ -0,0 +1,147 @@
+- [IDbox – Cost efficient device for self-sovereign identity](https://www.youtube.com/watch?v=h1Oz3oEtZxE)
+
+### Product Description
+
+- Smart contracts: See [GDocs Identity Terms Intent](https://docs.google.com/document/d/1K8tFjGneScuKiudpD3HfSpAd4eOe8012jn0Q3iDxEV0/edit#) and [Identity Readme](https://github.com/status-im/contracts/blob/develop/Identity.md)
+- Basic contracts demo: Helps engaging developers and bug hunting. 
+- Identity Sign-in: Actions for user creating or recovering their wallets.
+- Identity Usage: User may select their Identity to make tansactions in behalf of.
+- Identity Management: User may setup keys & recovery options for its identity, confirm claims and even update the contract code.
+
+### Requirements & Dependencies
+
+- ERC725 and ERC735 should be approved as standard and used in contracts interface, this swarm might influence how this standards end up defined, this is important for the solution working better with unknown future applications.
+
+- Idea [#150-gas-abstraction](https://github.com/status-im/ideas/pull/150) enables users to only use SNT* for everything in ethereum, or any other token/ether hold by Identity. *Only SNT can be used to gas relay deploy of Identity, after that user is free to choice.
+
+- Idea [#151-ens-usernames](https://github.com/status-im/ideas/pull/151) is important as a shortcut for user's identity address, otherwise QR-Codes or big hexadecimal strings would be needed.
+
+- Idea [152-friends-recovery](https://github.com/status-im/ideas/pull/152) is important to provide a first recovery contract option that uses friends to recover.
+
+## Dates
+
+### Proof-of-concept
+
+Goal Date: 2018-05-15
+
+Description: 
+- Working contracts system 
+- Basic UI for direct interaction with blockchain 
+
+### Minimal Viable Product
+
+Goal Date:  2018-06-01
+
+Description:
+- Status Wallet enables creation, association or recovery of Identity 
+- Status Wallet enables transactions to be executed by Identity
+- Identity management (basic)
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+### Seamless Status Integration
+
+Goal Date: 2018-06-15
+
+- Identity Sign-in:
+    - Create new;
+    - Reconfigure existing;
+- Identity Usage:
+    - Interact with other contracts in behalf of Identity;
+    - Sign claims about others Identity.
+- Identity Management 
+    - Keys: Add/Remove/Update 
+    - Accept Claims
+    - Update of Kernel
+    - Select a recovery contract
+    
+## Success Metrics
+
+Status Users opt-in using an Identity to enhance interaction with decetralize systems (such as Status itself). 
+Developers start using Status Identity implementation as base.
+New users adopt Status because Identity support, which will be required for many dapps in future.
+
+## Exit criteria
+
+Fully integrated on Status App.
```

Comment:
Would be useful to frame this in terms of specific capabilities provided in app

Reply:
Sure, good point, I'll elaborate on that,